Xamarin.Forms.Platform.Android/AppCompat/ButtonRenderer.cs

```diff
@@ -170,14 +187,20 @@ void UpdateBitmap()
 
 			var image = Context.Resources.GetDrawable(imageFile);
 
-			if (string.IsNullOrEmpty(Element.Text))
+			if (IsNullOrEmpty(Element.Text))
```

Comment:
Out of curiosity....why remove the qualifier? I think this makes it look like we wrote our own `IsNullOrEmpty` function on first glance.